etc/profile-m-z/quaternion.profile

```diff
@@ -0,0 +1,55 @@
+# Firejail profile for quaternion
+# Description: Desktop client for Matrix 
+# This file is overwritten after every install/update
+# Persistent local customizations
+include quaternion.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/Quotient/quaternion
+noblacklist ${HOME}/.config/Quotient
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/Quotient/quaternion
+mkdir ${HOME}/.config/Quotient
+whitelist ${HOME}/.cache/Quotient/quaternion
+whitelist ${HOME}/.config/Quotient
+whitelist ${DOWNLOADS}
+whitelist /usr/share/qt
+whitelist /usr/share/Quotient/quaternion
```

Comment:
Is `quaternion` the only thing in `/usr/share/Quotient`?

Reply:
Idk if you `yay -S quaternion` and `yay -S spectral-matrix-git` they'll end up in conflicted Qt libs ending with `quotient`,you can't use both at same time without purging the other. So I think they might share.

Reply:
Keep this?